delivery/client.py

```diff
@@ -25,6 +28,26 @@ class DeliveryServiceRoutes:
     def __init__(self, base_url: str):
         self._base_url = base_url
 
+    def auth(self):
+        return ci.util.urljoin(
+            self._base_url,
+            'auth',
+        )
+
+    def auth_configs(self):
+        return ci.util.urljoin(
+            self._base_url,
+            'auth',
+            'configs',
+        )
+
+    def openid_configuration(self):
```

Comment:
add ref to https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest in docstr